docs/extensibility/resources.md

```diff
@@ -62,11 +62,26 @@ The `list` section defines extra columns available in the list. The format is si
 
 - **path** - _[required]_ contains the path to the data used for the column.
 - **widget** - optional widget used to render the field referred to by the `path` property. By default the value is displayed verbatim. For more information about the available widgets, see [Display widgets](display-widgets.md).
+- **formula** - optional formula used to modify data referred to by the `path` property. In formula we use naming convention: `data.name` instead of just `name`. To see more details about using formulas see [JSONata](https://docs.jsonata.org/overview.html).
 
 ### Example
 
 ```json
-[{ "path": "spec.url" }, { "path": "spec.priority", "widget": "Badge" }]
+[
+  { "path": "spec.url" },
+  { "path": "spec.priority", "widget": "Badge" },
+  { "path": "spec.volumes", "formula": "$join(data.name, ', ')" },
+  {
+    "path": "spec.volumes",
+    "formula": "$filter(data, function ($v, $i, $a) {'configMap' in $keys($v)})", //List array of Volume objects that have config map"
+    //or
+    "formula": "data['configMap' in $keys($)]"
+  },
+  {
+    "path": "spec.volumes",
+    "formula": "$join(data['configMap' in $keys($)].name, ', ')" // List volume names of volumes that have config map
+  }
+]
 ```
```

Comment:
If you have an `OR` there, I'd suggest splitting this into two separate examples: 

``````suggestion
[
  { "path": "spec.url" },
  { "path": "spec.priority", "widget": "Badge" },
  { "path": "spec.volumes", "formula": "$join(data.name, ', ')" },
  {
    "path": "spec.volumes",
    "formula": "$filter(data, function ($v, $i, $a) {'configMap' in $keys($v)})", // List an array of Volume objects that have a config map"
  },
  {
    "path": "spec.volumes",
    "formula": "$join(data['configMap' in $keys($)].name, ', ')" // List volume names of volumes that have a config map
  }
]
```

or

```json
[
  { "path": "spec.url" },
  { "path": "spec.priority", "widget": "Badge" },
  { "path": "spec.volumes", "formula": "$join(data.name, ', ')" },
  {
    "path": "spec.volumes",
    "formula": "data['configMap' in $keys($)]"
  },
  {
    "path": "spec.volumes",
    "formula": "$join(data['configMap' in $keys($)].name, ', ')" // List volume names of volumes that have a config map
  }
]
```
``````

I'm not sure I split it up properly, so please correct me if I'm wrong.

Reply:
I changed it to sth like this:
```
{
  "path": "spec.volumes",
  "formula": "$filter(data, function ($v, $i, $a) {'configMap' in $keys($v)})", //List array of Volume objects that have config map"
},
{
  "path": "spec.volumes",  
  "formula": "data['configMap' in $keys($)]" //List array of Volume objects that have config map"
},
```